packages/@aws-cdk/core/lib/index.ts

```diff
@@ -30,7 +30,7 @@ export * from './cfn-json';
 export * from './removal-policy';
 export * from './arn';
 export * from './duration';
-export * from './from-cfn';
+export { ICfnFinder } from './from-cfn';
```

Comment:
We don't need to export this from this module anymore, right? Nothing is using it, unless I'm mistaken? So the entire `export * from './from-cfn';` line can be removed I think.